.github/workflows/reusable_testing.yml

```diff
@@ -28,12 +48,22 @@ jobs:
         # dependencies when migrating to other OS version.
         run: sudo dpkg -i tarantool*.deb
 
-      - name: Setup python3 for tests
+      - name: Setup Python for tests
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: ${{ matrix.python }}
+
+      - name: Install msgpack python package
+        if: matrix.msgpack-deps != ''
+        run: pip install ${{ matrix.msgpack-deps }}
 
       - name: Install connector requirements
+        # msgpack package is a replacement for deprecated msgpack-python.
+        # To test compatibility with msgpack-python we must ignore
+        # requirements.txt install, since it will install newest msgpack.
+        # Beware that if any new dependency will be added to requirements.txt,
+        # this step will be invalid.
```

Comment:
Maybe `sed -i -e 's/^msgpack.*$/${{ matrix.msgpack-deps }}/' requirements.txt` would be more reliable solution?

Reply:
Thank you, that's a really nice approach